nodejs/week2/meal-sharing/src/backend/api/reservation-router.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const router = express.Router();
+
+const reservations = require("./../data/reservations.json")
+
+router.get("/:id", async (request, response) => {
+  try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+      const realId = parseInt(request.params.id) - 1;
+      response.send(reservations[realId]);
```

Comment:
Here, the index is not necessarily the correct meal.

Reply:
for me it works good. would you give me the example thta it does not work. e.g. When I put http://localhost:3000/reservations/4 in the adrress bar it shows me reservation id 4.

Reply:
I made it correct. thanks